src/data_structures/fenwick.md

```diff
@@ -208,23 +180,23 @@ struct FenwickTreeMin {
 
     FenwickTreeMin(int n) {
```

Comment:
What does this code present then? Should we say segment tree can solve this problem for min ranges as min is associative?